com.core447.StreamController.yml

```diff
@@ -0,0 +1,99 @@
+id: com.core447.StreamController
+runtime: org.gnome.Platform
+runtime-version: '46'
+sdk: org.gnome.Sdk
+command: launch.sh
+
+finish-args:
+  - --socket=wayland
+  - --socket=fallback-x11
+  - --share=ipc
+  - --device=dri
+  - --filesystem=xdg-documents
+  - --share=network
+  - --socket=pulseaudio
+  - --device=all # Needed to communicate with the decks
+  - --talk-name=org.freedesktop.Flatpak # Allow plugins to run system commands
+  - --talk-name=org.freedesktop.UPower.*
+  - --talk-name=org.mpris.MediaPlayer2.*
+  - --talk-name=org.kde.kdeconnect.*
+  - --talk-name=org.gnome.Shell.*
+  - --talk-name=org.gnome.Shell.Extensions.*
```

Comment:
```suggestion
  - --talk-name=org.gnome.Shell.Extensions
```

Reply:
Seams like `org.gnome.Shell` is sufficient, I removed `org.gnome.Shell.Extensions` in 337b58fa31b3ef9da6cce795e705cbe1a20755a4